### ISSUE TITLE
Fix remaining accessibility issues

### ISSUE THREAD

**Issue by mvaivre** [alephium/desktop-wallet#383]
Accessibility issues:

* [x] Pressing enter on passphrase toggle when wallet and password are entered submits the form instead of expanding the section
* [x] Toggling the checkbox for "I understand that..." and then going to the passphrase input and typing doesnt work
* [x] Copy buttons are being tabbed to in the transaction rows
* [x] Double text being spoken (hidden text and the visible text) in the transaction rows
* [x] Offline mode icon not tabbable (should be so blind user can listen to screen reader and know wallet is offline...)

Everything else is more or less fine

_Originally posted by @LeeAlephium in https://github.com/alephium/desktop-wallet/issues/378#issuecomment-1261443029_
